test/DynamoCoreTests/AstBuilderTest.cs

```diff
@@ -81,25 +81,29 @@ public void TestSortNode2()
             var nodes = CurrentDynamoModel.CurrentWorkspace.Nodes.ToList();
 
             var shuffle = new ShuffleUtil<NodeModel>(nodes);
+            var id1 = "654c9a4b-3f58-4f90-9bde-c3e615100b12";
+            var id2 = "8a55ca22-4424-4ccb-bd2f-3fe79bbeaccf";
+            var id3 = "189689be-815b-4a6e-89cb-45b495962cca";
+            var id4 = "918ac1ed-98fa-457d-bdb7-fe7456ea3fb5";
 
             for (int i = 0; i < shuffleCount; ++i)
             {
                 var sortedNodes = AstBuilder.TopologicalSort(shuffle.ShuffledList).ToList();
                 Assert.AreEqual(sortedNodes.Count(), 4);
 
-                List<int> names = sortedNodes.Select(node => Int32.Parse(node.Name)).ToList();
+                List<string> ids = sortedNodes.Select(node => node.GUID.ToString()).ToList();
 
-                var nodePosMap = new Dictionary<int, int>();
-                for (int idx = 0; idx < names.Count; ++idx)
+                var nodePosMap = new Dictionary<string, int>();
+                for (int idx = 0; idx < ids.Count; ++idx)
                 {
-                    nodePosMap[names[idx]] = idx;
+                    nodePosMap[ids[idx]] = idx;
                 }
 
                 // no matter input nodes in whatever order, there invariants 
                 // should hold
```

Comment:
Would you mind updating this comment?  It appears a bunch and is strangely worded.  Otherwise, LGTM

Reply:
yep, can do.